src/Utilities/copy.jl

```diff
@@ -486,14 +454,6 @@ function copy_free_variables(
     return
 end
 
-function default_copy_to(dest::MOI.ModelLike, src::MOI.ModelLike)
-    Base.depwarn(
-        "default_copy_to(dest, src) is deprecated, use default_copy_to(dest, src, true) instead or default_copy_to(dest, src, false) if you do not want to copy names.",
-        :default_copy_to,
-    )
-    return default_copy_to(dest, src, true)
-end
```

Comment:
It seems crazy, but this has been sitting here since MOI 0.2.0! https://github.com/jump-dev/MathOptInterface.jl/commit/1ae8661f8a0151013c4d92ea1aeb66568e361e1e